docs/new-architecture-app-intro.md

```diff
@@ -201,3 +201,14 @@ You can implement the `jsExecutorFactoryForBridge:` method like this:
   );
 }
 ```
+
+## iOS: Setup Folly
+
+The previous step will incorporate in your iOS app a dependency called Folly. Folly requires some extra compiler flags to works properly. To set them up, follow these steps:
+
+1. In the Project Navigator (`cmd+1`), select your app project.
+1. In the **Targets** section, select the target with the name of your app.
+1. Select the **Build Settings** tab
+1. Search for _Other C++ Flags_
```

Comment:
Is it intentional that this is italic, while the others are bold?

Reply:
Yes, the bold things are titles or something clickable, while the italic is something the user has to type. It's fine to unify them with bold, if we prefer.

Reply:
No strong preference, I'm fine either way 👍